experimental/PropertyDDS/packages/property-inspector-table/tsconfig.json

```diff
@@ -1,11 +1,15 @@
 {
-	"extends": "@fluidframework/build-common/ts-common-config.json",
+	"extends": "../../../../common/build/build-common/tsconfig.bundler.json",
 	"compilerOptions": {
-		"jsx": "react",
 		"resolveJsonModule": true,
-		"typeRoots": ["../node_modules/@types"],
-		"outDir": "./dist",
+		"outDir": "./lib",
+		"types": ["jest"], // Required because of the comingled building of test and src files
 	},
 	"exclude": ["dist", "node_modules"],
-	"include": ["src/**/*", "assets/**/*", "custom.d.ts"],
+	"include": [
+		"src/**/*",
+		"assets/**/*",
+		"custom.d.ts",
+		"test/**/*", // Required because /src/InspectorTable.stories.tsx imports '../test/common.js'
```

Comment:
The comingling was pre-existing, but because the test sources aren't compiled (test files are *.js), we previously got away with reaching reaching back into '../test/*' without including "test/**/*".

Reply:
My impression from devtools-view was that stories were test files. So, the detangling would be to exclude stories from production build. But this week is my first experience with them.